app/views/layouts/application.html.erb

```diff
@@ -4,15 +4,15 @@
     <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
     <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
     <%= csrf_meta_tags %>
-    <title>Hogwarts
+    <title>Hogwarts</title>
   </head>
   <body>
     <nav>
       <h1>Welcome to Hogwarts!</h1>
-      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", student_path %> | <%= link_to "Enroll", student_path %></h2>
+      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", students_path %> | <%= link_to "Enroll", students_path %></h2>
```

Comment:
"View Students" and "Enroll" currently use the same path helper, but they should go to different pages.

What is the purpose of "Enroll"? What controller action should it map to? Use the answers to those questions to pick out the proper path helper from the output of `$ rails routes`